doc/tutorials/gapi/oak_devices/oak_devices.markdown

```diff
@@ -0,0 +1,38 @@
+Using DepthAI Hardware /OAK depth sensors {#tutorial_oak}
+=======================================================================================
+
+@tableofcontents
+
+@prev_tutorial{tutorial_gapi_face_beautification}
+
+![hardwares](pics/oak.jpg)
+
+Depth sensors compatible with DepthAI library are supported through OpenCV Graph API (or G-API) module. Depth map, RGB image and some other formats of output can be retrieved by using familiar interface of G-API module.
+
+In order to use DepthAI sensor with OpenCV you should do the following preliminary steps:
+-#  Install Intel DepthAI library/depthai-core (from here <https://github.com/luxonis/depthai-core>).
```

Comment:
Luxonis DepthAI library

Reply:
Why it has been resolved?

Reply:
@cabelo Intel -> Luxonis. Also it makes sense to use markdown notation for the link.